app/views/statuses/_task_fields.html.erb

```diff
@@ -9,13 +9,13 @@
         <div class="col-4">
             <div class="form-outline">
                 <%= f.label :git_pr_link, "Git Pr", class: "form-label" %>
-                <%= f.text_field :git_pr_link, type:"url", class: "form-control" %>
+                <%= f.url_field :git_pr_link, type:"url", class: "form-control" %>
```

Comment:
Use single quote when no string interpolation and special characters. Add space after : `type:"url"` Here add space after :